zerocopy-derive/tests/include.rs

```diff
@@ -26,7 +26,7 @@ mod imp {
     #[allow(unused)]
     pub use {
         ::core::{
-            assert_eq,
+            assert_eq, assert_ne,
```

Comment:
Did rustfmt perhaps miss this file? My intuition is that these two names should be on different lines.

Reply:
Surprised me too but that's how it seems to autoformat

Reply:
(If you put them on different lines it merges them)